learn-regular-expressions-by-building-a-spam-filter/index.js

```diff
@@ -0,0 +1,40 @@
+const messageInput = document.getElementById("message-input");
+const result = document.getElementById("result");
+const checkMessageButton = document.getElementById("check-message-btn");
+
+/**
+ * I read through the previous discussions for this project
+ * and we can start with matches for literal phrases and then have the
+ * campers create more complex regex which seemed to be the original intent for this project.
+ *
+ *
+ *
+ */
+
+const blacklistRegexps = [
+  /[0-9] hundred|thousand|million|billion dollars/gi,
+  /PLEASE HELP|ASSIST ME/i,
```

Comment:
This would be a great expression to start with.

If we want to teach how to match a whole phrase like "please help me", this would be a good place to introduce a capture group around `HELP|ASSIST`.

Reply:
I can add a comment in the codebase about adding a step to refactor this to include the capturing group